src/sql/workbench/services/objectExplorer/browser/objectExplorerViewTreeShim.ts

```diff
@@ -26,6 +26,7 @@ export const IOEShimService = createDecorator<IOEShimService>(SERVICE_ID);
 export interface IOEShimService {
 	_serviceBrand: undefined;
 	getChildren(node: ITreeItem, viewId: string): Promise<ITreeItem[]>;
+	refreshNode(node: ITreeItem, viewId: string): Promise<ITreeItem[]>;
```

Comment:
I wouldn't have a separate function here, this should all just be wrapped up in getchildren.

Reply:
I'm fine with that - just wasn't sure what the contract was for this service. If you think it's fine I can make that change since that seems to be the assumption that the trees generally have anyways.